From cd809c2b04f9eb3c69013c73d444b6fd44d53de4 Mon Sep 17 00:00:00 2001
From: Ribesg <Ribesg@yahoo.fr>
Date: Fri, 4 Apr 2014 20:09:10 +0200
Subject: [PATCH] This is Nukkit.

* Edit README.md
* Edit pom.xml
* Change name of Server implementation

Modified files:
* README.md
* pom.xml
* src/main/java/org/bukkit/craftbukkit/CraftServer.java

diff --git a/README.md b/README.md
index cda766b..4f3686f 100644
--- a/README.md
+++ b/README.md
@@ -1,18 +1,18 @@
-CraftBukkit
-===========
+![Nukkit](http://www.ribesg.fr/images/nukkit.png)
 
-A Bukkit (Minecraft Server API) implementation
+Nukkit
+===========
 
-Website: [http://bukkit.org](http://bukkit.org)  
-Bugs/Suggestions: [http://leaky.bukkit.org](http://leaky.bukkit.org)  
-Contributing Guidelines: [CONTRIBUTING.md](https://github.com/Bukkit/CraftBukkit/blob/master/CONTRIBUTING.md)
+* Basically CraftBukkit with stuff
+* Will only be updated to CraftBukkit Beta and Recommended Builds
 
-Compilation
------------
+Yet Another Fork of CraftBukkit
+-------------------------------
+I'm doing this mainly for me. I like Spigot but **IN MY OPINION**:
 
-We use maven to handle our dependencies.
+* it's a bit unstable, it doesn't crash often, but it still crashed at least once in 2 months
+* it has too many things/features in it that I don't need
+* the only real advantage it has over CraftBukkit is the integrated Orebfuscator
 
-* Install [Maven 3](http://maven.apache.org/download.html)
-* Check out and install [Bukkit](http://github.com/Bukkit/Bukkit)
-    * *Note*: this is not needed as the repository we use has Bukkit too, but you might have a newer one (with your own changes :D)
-* Check out this repo and: `mvn clean package`
+I decided to just make my own simple fork. I don't know what I will add to it.
+There's no plan, and there's no support. Nukkit is under GPLv3.
diff --git a/pom.xml b/pom.xml
index ab24e56..2dc921f 100644
--- a/pom.xml
+++ b/pom.xml
@@ -1,12 +1,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.bukkit</groupId>
-  <artifactId>craftbukkit</artifactId>
+  <groupId>fr.ribesg.nukkit</groupId>
+  <artifactId>nukkit</artifactId>
   <packaging>jar</packaging>
   <version>1.7.9-R0.1</version>
-  <name>CraftBukkit</name>
-  <url>http://www.bukkit.org</url>
+  <name>Nukkit</name>
+  <url>http://www.ribesg.fr</url>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
@@ -14,29 +14,16 @@
     <junit.version>4.11</junit.version>
     <minecraft.version>1.7.9</minecraft.version>
     <minecraft_version>1_7_R3</minecraft_version>
-    <buildtag.prefix>git-Bukkit-</buildtag.prefix>
+    <buildtag.prefix>git-Nukkit-</buildtag.prefix>
     <buildtag.suffix></buildtag.suffix>
   </properties>
 
   <scm>
-    <connection>scm:git:git://github.com/Bukkit/CraftBukkit.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/Bukkit/CraftBukkit.git</developerConnection>
-    <url>https://github.com/Bukkit/CraftBukkit</url>
+    <connection>scm:git:git://github.com/Nukkit/Nukkit.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com/Nukkit/Nukkit.git</developerConnection>
+    <url>https://github.com/Nukkit/Nukkit</url>
   </scm>
 
-  <distributionManagement>
-    <repository>
-      <id>repobo-rel</id>
-      <name>repo.bukkit.org Releases</name>
-      <url>http://repo.bukkit.org/content/repositories/releases/</url>
-    </repository>
-    <snapshotRepository>
-      <id>repobo-snap</id>
-      <name>repo.bukkit.org Snapshots</name>
-      <url>http://repo.bukkit.org/content/repositories/snapshots/</url>
-    </snapshotRepository>
-  </distributionManagement>
-
   <repositories>
     <repository>
       <id>repobo-snap</id>
@@ -178,9 +165,9 @@
           <archive>
             <manifestEntries>
               <Main-Class>org.bukkit.craftbukkit.Main</Main-Class>
-              <Implementation-Title>CraftBukkit</Implementation-Title>
+              <Implementation-Title>Nukkit</Implementation-Title>
               <Implementation-Version>${describe}</Implementation-Version>
-              <Implementation-Vendor>Bukkit Team</Implementation-Vendor>
+              <Implementation-Vendor>Nukkit Team</Implementation-Vendor>
               <Specification-Title>Bukkit</Specification-Title>
               <Specification-Version>${api.version}</Specification-Version>
               <Specification-Vendor>Bukkit Team</Specification-Vendor>
diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index e6c8575..30389a9 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -188,7 +188,7 @@ import com.google.common.collect.MapMaker;
 import jline.console.ConsoleReader;
 
 public final class CraftServer implements Server {
-    private final String serverName = "CraftBukkit";
+    private final String serverName = "Nukkit";
     private final String serverVersion;
     private final String bukkitVersion = Versioning.getBukkitVersion();
     private final Logger logger = Logger.getLogger("Minecraft");
--
